server/form-pages/apply/access-and-healthcare/accessNeedsMobility.test.ts

```diff
@@ -32,7 +32,7 @@ describe('AccessNeedsMobility', () => {
     })
   })
 
-  itShouldHaveNextValue(new AccessNeedsMobility({}, application), '')
+  itShouldHaveNextValue(new AccessNeedsMobility({}, application), 'covid')
```

Comment:
Should we have a test for if additionalNeeds includes `mobility`?

Reply:
I could be getting the wrong end of the stick here but I think we do here: https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/5d5ae90646ac9e711686df1d0b70f66f6b307d23/server/form-pages/apply/access-and-healthcare/accessNeeds.test.ts#L44

Reply:
Sorry, looking at the wrong test! 😆